tests/migration_test.go

```diff
@@ -1001,6 +1002,29 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 			})
 
+			It("should migrate vmi with a usb disk", func() {
+
+				vmi := libvmi.NewAlpineWithTestTooling(
+					libvmi.WithEmptyDisk("disk0", v1.DiskBusUSB, resource.MustParse("128Mi")),
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				)
+
+				By("Starting the VirtualMachineInstance")
+				vmi = runVMIAndExpectLaunch(vmi, 240)
```

Comment:
@acardace this function doesn't exist anymore, which is why the tests fail

Reply:
thanks, didn't rebase against master in my local workspace :)